diff --git a/dist/chunks/lib-q6ns0vZr.js b/dist/chunks/lib-q6ns0vZr.js
index fe0d0c62c9a4159f17437d44e8d128db4bea2ae4..42fe54a9ed742c4082aceef193dadb7a6784e1f3 100644
--- a/dist/chunks/lib-q6ns0vZr.js
+++ b/dist/chunks/lib-q6ns0vZr.js
@@ -819,7 +819,9 @@ function esmShimPlugin() {
                 if (code.includes(CJSShim) || !CJSyntaxRe.test(code)) {
                     return null;
                 }
-                const lastESMImport = findStaticImports(code).pop();
+                const lastESMImport = this.parse(code)
+                    .body.filter(node => node.type === 'ImportDeclaration')
+                    .pop();
                 const indexToAppend = lastESMImport ? lastESMImport.end : 0;
                 const s = new MagicString(code);
                 s.appendRight(indexToAppend, CJSShim);
